examples/gno.land/p/demo/simpledao/propstore.gno

```diff
@@ -59,19 +67,24 @@ func (p *proposal) IsExpired() bool {
 	return false // this proposal never expires
 }
 
-func (p *proposal) Render() string {
+func (p *proposal) Render(idx int, authorUsername string) string {
 	// Fetch the voting stats
 	stats := p.Stats()
 
-	output := ""
-	output += ufmt.Sprintf("Author: %s", p.Author().String())
-	output += "\n\n"
-	output += p.Description()
-	output += "\n\n"
-	output += ufmt.Sprintf("Status: %s", p.Status().String())
-	output += "\n\n"
+	output := ufmt.Sprintf("# Proposal #%d - %s\n\n", idx, p.title)
```

Comment:
Can you make these concatenations nicer? Maybe a string builder.

Reply:
It doesn't make too much of a difference, imho; but I did it:

[090966a](https://github.com/gnolang/gno/pull/3096/commits/090966a56814973e8b890b8792e1825d552e1de1) & [7dca498](https://github.com/gnolang/gno/pull/3096/commits/7dca49871ae5acfeaf7f5016a78800249c3990ed)